src/components/projects/lean-workflow/CommentList.tsx

```diff
@@ -121,11 +131,68 @@ export const CommentList: React.FC<Props> = ({ selected, editor }) => {
       if (highlight) {
         // newComment.originalText = getHighlightText(highlight, state)
         newComment.originalText = highlight.text
-        setNewComment(newComment)
       }
+      setNewComment(newComment)
     }
   }, [commentTarget, doc, newComment, state, currentUser])
 
+  const commentsLabel = useMemo(() => {
+    let labels: { [p: string]: string } = {}
```

Comment:
Perhaps we can use Map here? I see it is unpacked on line 145 in the descendants iteration but it is not exactly clear to me what is the value of avoiding mutating it. It is of course not critical.

Reply:
oh that makes sense to use a map